packages/Button/src/Button.spec.tsx

```diff
@@ -16,8 +16,20 @@ describe('packages/Button', () => {
     </Button>,
   );
 
+  function isElement(el: Node): el is HTMLElement {
+    return el.nodeType === Node.ELEMENT_NODE;
+  }
+
+  function isButton(el: Node): el is HTMLButtonElement {
+    return isElement(el) && el.tagName.toLowerCase() === 'button';
+  }
```

Comment:
These helper typeguards could probably be extracted somewhere, but I'm not sure where that should be

Reply:
So I haven't seen this `is` syntax yet. If I'm understanding correctly, the `(type): type is someType` syntax is interpreted as: "if type is someType, this function will return true".

If so, this is sweet.

Reply:
Also, we can probably break these out into `lib` at a later time (I think we talked a bit about this in the channel?).

Reply:
Yup, and yes! [Docs for User-Defined Type Guards](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards)